Provider/ProviderInterface.php

```diff
@@ -6,29 +6,9 @@
 
 interface ProviderInterface
 {
-    /**
-     * @return SitemapSection
-     */
-    public function getSection();
-
-    /**
-     * @return string
-     */
-    public function getSectionName();
-
-    /**
-     * @return int
-     */
-    public function getNumberOfPages();
-
-    /**
-     * @return int
-     */
-    public function getCount();
-
-    /**
-     * @param int $pageNumber
-     * @return SitemapSectionPage
-     */
-    public function getPage($pageNumber);
+    public function getSection(): SitemapSection;
```

Comment:
Can we add a newline between every method?

Reply:
I will implement CS when this is merged and fix everything 👍